quint/io-cli-tests.md

```diff
@@ -864,7 +864,7 @@ rm out-itf-example.itf.json
 
 <!-- !test in run with n-traces itf -->
 ```
-quint run --out-itf=out-itf-example.itf.json --n-traces=3 --max-steps=5 --seed=123  ../examples/tutorials/coin.qnt
+quint run --out-itf='out-itf-example{seq}.itf.json' --n-traces=3 --max-steps=5 --seed=123  ../examples/tutorials/coin.qnt
```

Comment:
Do we actually need to update this? Isn't the behavior on the old version of the test still the default?

Reply:
My bad, forgot to revert this!

Reply:
6988f0e0